test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1266,6 +1266,38 @@ static public void testCreateLargeNumberOfPoint2D() throws Throwable {
 		}
 	}
 
+	/*
+	 * Create a value type and read the fields before
+	 * they are set. The test should Verify that the
+	 * flattenable fields are set to the default values.
+	 * NULL should never be observed for Qtypes.
+	 */
+	@Test(priority=1)
+	static public void testDefaultValues() throws Throwable {
```

Comment:
test it with other valuettypes, ie. assortedvalueobjectwithlongalignment and assortedrefobjectwithlongalignment